addons/web/models/ir_http.py

```diff
@@ -17,6 +17,7 @@ class Http(models.AbstractModel):
 
     def webclient_rendering_context(self):
         return {
+            'get_modules_order': lambda: json.dumps(module_boot()),
```

Comment:
why this change?

Reply:
This order was previously given to the views in `ir_qweb.py` in the evaluation context when rendering views. Now that assets are not generated by views anymore we have to find another way to give the order to the webclient